Makefile

```diff
@@ -1907,6 +1909,24 @@ hewiki_tuning_reports: \
 	tuning_reports/hewiki.goodfaith.md
 
 
+############################# Hindi Wikipedia ################################
+# From https://quarry.wmflabs.org/query/44835
+datasets/hiwiki.sampled_revisions.10k_2020.json:
+	wget -qO- https://quarry.wmflabs.org/run/467016/output/0/json-lines > $@
+
+datasets/hiwiki.autolabeled_revisions.10k_2020.json: \
+		datasets/hiwiki.sampled_revisions.10k_2020.json
+	cat $< | \
+	./utility autolabel --host=https://hi.wikipedia.org \
+		--trusted-groups=sysop,oversight,bot,rollbacker,checkuser,abusefilter,bureaucrat,autopatrolled,autoreviewer,reviewer \
```

Comment:
Shouldn't `autoreviewer` be replaced by `autoreview`? 
https://hi.wikipedia.org/wiki/Special:ListUsers?group=autoreview&uselang=en

Reply:
Good catch.  Just pushed a fix and luckily it didn't change the data I pushed to Wikilabels.  I'm guessing autoreview has enough overlap with another group that it didn't matter.